queries.sql

```diff
@@ -7,4 +7,26 @@ SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered = 't';
 SELECT * from animals WHERE name != 'Gabumon';
 SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
- 
\ No newline at end of file
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```

Comment:
- [ ] Same case as above. 👇🏼 

  > - After the roll back verify if all records in the animals table still exist. After that you can start breath as usual ;)